### ISSUE TITLE
Searching users

### ISSUE THREAD

**Issue by KujinKrush** [codewars/codewars.com#2202]
I believe that we should be able to search users because I can't friend my family members because I can't find them and others are trying to search some of their friends.

**Comment by Blind4Basics** [comment]
Yes. Temporary solution: you can all define the same clan's name, then you'll automatically be grouped.

Other solution: you send each others the links to your profiles to follow each others.

**Comment by FArekkusu** [comment]
Duplicate of #701.

**Comment by KujinKrush** [comment]
Thanks
________________________________

**Comment by Callmesauce** [comment]
let's be friends.

**Comment by Tallons** [comment]
Another solutions is if you know your friend's username,  then just add it to the following URL and add your friend's username as the parameter. After navigating to their profile, click on the follow button.

URL:
[https://codewars.com/users/{username](https://codewars.com/users/{ username here }

Example:
Username: Tallons
https://codewars.com/users/Tallons
